x-pack/test/security_solution_cypress/es_archives/exceptions/data.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
For the modal cypress tests, it was unnecessary to load a large file such as auditbeat, needed a small simple nested a non-nested field to test with is all.